src/users/users.controller.ts

```diff
@@ -376,4 +379,31 @@ export class UsersController {
       telegram: dto.telegram,
     });
   }
+
+  @ApiExcludeEndpoint()
+  @Post(':id/token')
+  @UseGuards(ApiKeyGuard)
+  async createAuthToken(
+    @Param('id', new IntIsSafeForPrismaPipe())
+    id: number,
+  ): Promise<SerializedUserLoginUrl> {
+    const user = await this.usersService.findOrThrow(id);
+
+    const secret: Secret = this.config.get<string>('JWT_TOKEN_SECRET');
+    const options: SignOptions = {
+      algorithm: 'HS256',
+      expiresIn: '1 days',
```

Comment:
nit: because 1 and plural days reads a little weird
```suggestion
      expiresIn: '1d',
```